pages/Category.vue

```diff
@@ -100,9 +100,8 @@
                 :score-rating="product.rating.score"
                 :link="product.link"
                 link-tag="a"
-                :is-on-wishlist="isOnWishlist(product.data)"
+                :wishlist-icon="false"
                 class="products__product-card"
-                @click:wishlist="toggleWishlist(product.data)"
```

Comment:
`product.data` key from `prepareCategoryProduct` computed property can be removed.

Reply:
done